### PR TITLE
Adding doc for setting the dns domain for staging

### PR DESCRIPTION
**Pull request by goldblatt**
### Description
adding doc for setting the dns domain for staging

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ x] Extended the README / documentation, if necessary

### Issue(s)
Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

**Review by zaralouis-sf — APPROVED** [review]

File: cloud/deploys/staging_azure/README.md [hunk 1]
```diff
@@ -109,3 +109,12 @@ az keyvault secret set --name app-secret-key --vault-name [key vault name] --val
 ```
 
 Then, in order to use the Key Vault as a data source in Terraform, set the `key_vault_name` variable in your `auto.tfvars` file to the name of the key vault and the `key_vault_resource_group` to the resource group the key vault is in.
+
+## Configuring the Domain to be staging
```
Comment (bion) [inline cloud/deploys/staging_azure/README.md:113]: nit:
```suggestion
## Configuring the staging domain
```

File: cloud/deploys/staging_azure/README.md [hunk 2]
```diff
@@ -109,3 +109,12 @@ az keyvault secret set --name app-secret-key --vault-name [key vault name] --val
...
+The terraform script configures the azure app service to be routed to the staging name, but you will need to manually add a cname and txt configuration to your domain provider (e.g https://domains.google.com).
```
Comment (bion) [inline cloud/deploys/staging_azure/README.md:114]: > to be routed to the staging name
which staging name?

File: cloud/deploys/staging_azure/README.md [hunk 3]
```diff
@@ -109,3 +109,12 @@ az keyvault secret set --name app-secret-key --vault-name [key vault name] --val
...
 Then, in order to use the Key Vault as a data source in Terraform, set the `key_vault_name` variable in your `auto.tfvars` file to the name of the key vault and the `key_vault_resource_group` to the resource group the key vault is in.
+
+## Configuring the Domain to be staging
+The terraform script configures the azure app service to be routed to the staging name, but you will need to manually add a cname and txt configuration to your domain provider (e.g https://domains.google.com). 
+
+To do that add the custom records via the domain provider webiste. 
+1) CNAME record which points from the 'staging-azure.civiform.dev' to the hostname that gets generated from terraform (you can find this from the terraform output or via the azure portal)
+2) TXT record with key 'asuid.staging-azure.civiform.dev' and value that matches the custom domain verification id in the azure portal (you can find this by navigating to the custom domains in the app service setting). 
+
+Note it should take a few minutes to propogate
```
Comment (bion) [inline cloud/deploys/staging_azure/README.md:120]: nit:
```suggestion
Note it should take a few minutes to propagate.
```

**Review by bion — APPROVED** [review]
a few minor comments

File: cloud/deploys/staging_azure/README.md (hunk 2 above)
Comment (goldblatt) [reply cloud/deploys/staging_azure/README.md]: alright clarified!

**Pull request merged**